docs/sources/reference/components/loki/loki.source.podlogs.md

```diff
@@ -231,6 +231,21 @@ cluster to distribute the load of log collection between all cluster nodes.
 If {{< param "PRODUCT_NAME" >}} is _not_ running in clustered mode, then the block is a no-op and
 `loki.source.podlogs` collects logs based on every PodLogs resource discovered.
 
+Clustering only looks at the following labels for determining the shard key:
```

Comment:
```suggestion
Clustering looks at the following labels for determining the shard key:
```

We can drop the only. If you think it is still needed for clarity, then switch its location to read `Clustering looks only at the...`